api/src/main/java/org/apache/cloudstack/api/response/ManagementServerResponse.java

```diff
@@ -35,12 +38,68 @@
 
     @SerializedName(ApiConstants.STATE)
     @Param(description = "the state of the management server")
-    private ManagementServerHost.State state;
+    private State state;
 
     @SerializedName(ApiConstants.VERSION)
     @Param(description = "the version of the management server")
     private String version;
 
+    @SerializedName(ApiConstants.JAVA_DISTRIBUTION)
+    @Param(description = "the java distribution name running the management server process")
+    private String javaDistribution;
+
+    @SerializedName(ApiConstants.JAVA_VERSION)
+    @Param(description = "the version of the java distribution running the management server process")
+    private String javaVersion;
+
+    @SerializedName(ApiConstants.OS_DISTRIBUTION)
+    @Param(description = "the name of the OS distribution running on the management server")
+    private String osDistribution;
+
+    @SerializedName(ApiConstants.LAST_START)
+    @Param(description = "the last time this Management Server was started")
+    private Date lastStart;
+
+    @SerializedName(ApiConstants.LAST_STOP)
+    @Param(description = "the last time this Management Server was started")
```

Comment:
typo, started => _stopped_

```suggestion
    @Param(description = "the last time this Management Server was stopped")
```